src/drive/web/modules/queries.js

```diff
@@ -325,6 +325,38 @@ export const buildEncryptionByIdQuery = id => ({
   }
 })
 
+/**
+ * Provide selector and options to fetch files
+ *
+ * @param {string} bookmark - The query bookmark
+ * @returns {{options: {MangoQueryOptions}, selector: object}} A minimalist file list
+ */
+export const prepareSuggestionQuery = (bookmark = '') => {
+  const selector = {
+    _id: {
+      $gt: null
+    }
+  }
+  const options = {
+    partialFilter: {
+      _id: {
+        $ne: TRASH_DIR_ID
+      },
+      path: {
+        $or: [{ $exists: false }, { $regex: '^(?!/.cozy_trash)' }]
+      },
+      trashed: {
+        $or: [{ $exists: false }, { $eq: false }]
+      }
+    },
+    fields: ['_id', 'trashed', 'dir_id', 'name', 'path'],
```

Comment:
No need for the file mime, or class? And type, to know when it's a directory?

Reply:
type has been added as a fields

mime or class are not needed.